### PR TITLE
Apply filter after TADIR selection 

### PR DESCRIPTION
**Pull request by fmabap**
#6428
This PR is the optimization for "stage by transport"

### CODE REVIEW

File: src/objects/core/zcl_abapgit_tadir.clas.abap [hunk 1]
```diff
@@ -384,6 +384,9 @@ CLASS zcl_abapgit_tadir IMPLEMENTATION.
   METHOD zif_abapgit_tadir~read.
 
     DATA: li_exit TYPE REF TO zif_abapgit_exit.
+    DATA: lr_tadir TYPE REF TO zif_abapgit_definitions=>ty_tadir.
+    DATA: lo_filter TYPE REF TO zcl_abapgit_repo_filter.
```
Comment (larshp) [inline src/objects/core/zcl_abapgit_tadir.clas.abap:388]: `lo_filter` is not used, and can be deleted?

**Review by larshp — CHANGES_REQUESTED** [review]

**Comment by larshp** [comment]
@fmabap please update the description, this is the optimization for "stage by transport" ?

**Comment by fmabap** [comment]
> @fmabap please update the description, this is the optimization for "stage by transport" ?

I updated the description of the PR

**Review by larshp — APPROVED** [review]
👍

**Pull request merged**